Microsoft.Research/RegressionTest/ClousotTests/AsyncTestDriver.cs

```diff
@@ -20,147 +9,146 @@
+        public static readonly uint MaxWaitHandles_AllButOne = Math.Max(1, (uint)(Environment.ProcessorCount - 1));
 
-    private readonly Action<Options, Output> action;
-    private readonly IsolatedAction<Options> actionDelegate;
-    private readonly Func<Options, bool> skipTest;
-    private Dictionary<string, IAsyncResult> testAsyncResults;
-    private readonly uint maxWaitHandles;
-    private WaitHandle[] waitHandles;
-    private int nbWaitHandles;
-    private bool beginTestsProcessed = false;
-    private bool orderReversed = false;
+        private static readonly int SingleTestMaxWait = 200000;
 
-    public string BeginMessage;
+        private readonly Action<Options, Output> action;
+        private readonly IsolatedAction<Options> actionDelegate;
+        private readonly Func<Options, bool> skipTest;
+        private Dictionary<string, IAsyncResult> testAsyncResults;
+        private readonly uint maxWaitHandles;
+        private WaitHandle[] waitHandles;
+        private int nbWaitHandles;
+        private bool beginTestsProcessed = false;
+        private bool orderReversed = false;
 
-    public AsyncTestDriver(Action<Options, Output> action, Func<Options, bool> skipTest)
-      : this(action, skipTest, MaxWaitHandles_Default)
-    { }
+        public string BeginMessage;
 
-    public AsyncTestDriver(Action<Options, Output> action, Func<Options, bool> skipTest, uint maxWaitHandles)
-    {
-      this.action = action;
-      this.actionDelegate = this.ActionAsIsolated;
-      this.skipTest = skipTest;
-      this.maxWaitHandles = maxWaitHandles;
-    }
+        public AsyncTestDriver(Action<Options, Output> action, Func<Options, bool> skipTest)
+          : this(action, skipTest, MaxWaitHandles_Default)
+        { }
 
-    // We have no control on the order of the tests, so we make sure
-    // to always call Begin before End
-
-    public void BeginTest(Options options)
-    {
-      if (this.skipTest(options))
-        return;
+        public AsyncTestDriver(Action<Options, Output> action, Func<Options, bool> skipTest, uint maxWaitHandles)
+        {
+            this.action = action;
+            actionDelegate = this.ActionAsIsolated;
+            this.skipTest = skipTest;
+            this.maxWaitHandles = maxWaitHandles;
+        }
 
-      this.beginTestsProcessed = true;
+        // We have no control on the order of the tests, so we make sure
+        // to always call Begin before End
 
-      if (this.orderReversed)
-        this.EndTestInternal(options);
-      else
-        this.BeginTestInternal(options);
-    }
+        public void BeginTest(Options options)
+        {
+            if (skipTest(options))
+                return;
 
-    public void EndTest(Options options)
-    {
-      if (this.skipTest(options))
-        return;
+            beginTestsProcessed = true;
 
-      if (!this.beginTestsProcessed)
-        this.orderReversed = true;
+            if (orderReversed)
```

Comment:
No, if the `if` statement requires braces then the `else` statement must also use braces (and vice-versa). This will eventually be enforced by DotNetAnalyzers/StyleCopAnalyzers#716 (which provides document-, project-, and solution-wide automatic fixes), whether or not codeformatter.exe handles it in this pull request.